.github/workflows/crio.yml

```diff
@@ -55,28 +48,38 @@ jobs:
           ginkgo version
           sudo cp $(command -v ginkgo) /usr/local/bin
 
-      - name: Install CRI-O latest main
+      - name: Install CRI-O
         run: |
           curl https://raw.githubusercontent.com/cri-o/packaging/main/get | sudo bash
 
-      - name: Configure and start CRI-O
+      - name: Configure CRI-O
         run: |
           sudo mkdir -p /etc/crio/crio.conf.d
           printf '[crio.runtime]\nlog_level = "debug"\n' | sudo tee /etc/crio/crio.conf.d/01-log-level.conf
-          printf '[crio.runtime]\nseccomp_use_default_when_empty = false\n' | sudo tee /etc/crio/crio.conf.d/02-seccomp.conf
+
+      - name: Configure CRI-O to use conmon-rs intead of the default conmon
+        if: ${{matrix.monitor == 'conmon-rs'}}
+        run: |
+          sudo sed -i -E 's;(monitor_path = ).*;\1"/usr/libexec/crio/conmonrs"\nruntime_type = "pod";g' /etc/crio/crio.conf.d/10-crio.conf
```

Comment:
I wonder whether it might be simpler to drop a drop-in file called `99-crio.conf` that has the properties we want to override.

Thoughts?

Reply:
It depends, that way I can just change one value by relying on the shipped defaults and easily verify everything manually in the "Show the CRI-O config drop-in" step. I think in the end it doesn't matter much.